docs/app/Examples/elements/List/Types/ListDivided.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { List } from 'stardust'
+
+const ListDivided = () => {
+  return (
+    <List divided relaxed>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
+        <List.Content>
+          <List.Header as='a'>Semantic-Org/Semantic-UI</List.Header>
+          <List.Description as='a'>Updated 10 mins ago</List.Description>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
+        <List.Content>
+          <List.Header as='a'>Semantic-Org/Semantic-UI-Docs</List.Header>
+          <List.Description as='a'>Updated 22 mins ago</List.Description>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
```

Comment:
Agreed.  I was on the fence as well, but after sleeping on it and hearing this I think we should stick with the API pattern we've chosen rather than make an exception.

@layershifter let's keep it consistent and go with `List.Icon`.  Thanks for the input guys.